Tools-Override/CloudTest.targets

```diff
@@ -429,7 +429,31 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="CompressRuntimeDir" Condition="'$(SkipArchive)' != 'true'">
+  <Target Name="CopyCoreCLRTestLauncherForUAPToRuntimeDir" Condition="'$(BuildingUAPVertical)' == 'true'">
+    <PropertyGroup>
+      <_LauncherFolderCopy>$(TestSharedFxDir)/Launcher/</_LauncherFolderCopy>
+      <_RunnerFolderCopy>$(TestSharedFxDir)/Runner/</_RunnerFolderCopy>
+    </PropertyGroup>
+    <ItemGroup>
+      <UAPLauncherFolderFiles Include="$(UAPLauncherFolder)/**/*" />
```

Comment:
I think it would be better to just give a message saying "skipping test run because blah isn't set".